### ISSUE TITLE
Kibana Version Support 5.6.9

### ISSUE THREAD

**Issue by Infraded** [ppadovani/KibanaNestedSupportPlugin#66]
Keeping 5.6.x alive :D

Thanks

**Comment by ppadovani** [comment]
I'll do this as soon as I've fixed the latest issues reported, as they affect the 5.6 releases.

**Comment by ppadovani** [comment]
Ok... got the latest issues in 6.2.2/6.2.3/6.2.4 fixed. What I'm going to do is start back porting the fixes, and release each as I go. I'll eventually get to 5.6.7 fix that, then create releases for 5.6.8 and 5.6.9. I'm surprised they are still creating new releases for the 5.6.x release.

**Comment by Infraded** [comment]
No rush on the update and thanks for all your effort. If you're curious about the 5.x support, they have a full table here: https://www.elastic.co/support/eol

Tldr: 5.6.x has support from Elastic until March 2019. They have an 18 month lifecycle for major versions, with the last minor version being supported until the 2nd major version after is released (7.x in this case).

**Comment by ppadovani** [comment]
Complete
